org.mixedrealitytoolkit.spatialmanipulation/BoundsControl/BoundsHandleInteractable.cs

```diff
@@ -161,6 +163,23 @@ public void HideOnStartup()
             wasOccludedLastFrame = true;
         }
 
+        /// <summary>
+        /// Sets <see cref="IsOccluded"/> to true, and forces handling of occlusion immediately."/>
```

Comment:
Sorry this is probably my fault... remove this bogus stuff at the end of the sentence...

 "/>